viewer/lib/worldView.js

```diff
@@ -54,13 +55,20 @@ class WorldView extends EventEmitter {
         this.emitter.emit('entity', { id: e.id, name: e.name, pos: e.position, width: e.width, height: e.height, username: e.username })
       }
     }
+
+    this._timecycleUpdateInterval = setInterval(() => {
```

Comment:
I am not sure about this. Shouldn't it be controlled by the client ticks ? eg mineflayer